app/src/main/java/com/example/parkingsystemkotlin/fragments/DateReservationDialogFragment.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.parkingsystemkotlin.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.example.parkingsystemkotlin.databinding.FragmentDialogDatePickerBinding
+import com.example.parkingsystemkotlin.mvp.contract.DialogDateResarvationContract
+import com.example.parkingsystemkotlin.mvp.presenter.DialogDateResarvationPresenter
+import com.example.parkingsystemkotlin.mvp.view.DialogDateResarvationView
+import com.example.parkingsystemkotlin.utils.Constant
+
+class DateReservationDialogFragment : DialogFragment() {
+    private lateinit var binding: FragmentDialogDatePickerBinding
+    private lateinit var presenter: DialogDateResarvationContract.DialogFragmentDateReservationPresenter
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentDialogDatePickerBinding.inflate(layoutInflater)
+        arguments?.let {
```

Comment:
are you using the arguments here? if not, delete this pls

Reply:
Done

Reply:
Done